workers/Dockerfile.tests

```diff
@@ -0,0 +1,3 @@
+FROM dr_worker
```

Comment:
It is. I was trying to trim down lines with this change but it doesn't work at all. I basically wanted a new dockerfile with just a different entrypoint, but doing it like this won't force a rebuild of dr_worker if the code changes, so the code stays stale. My latest change should fix this (I just go back to how this file was before.)